test/DBServer.cpp

```diff
@@ -212,7 +210,7 @@ int main(){
 		std::cout << "Installing federation role" << std::endl;
 		auto res = runCommand("kubectl",
 		                      {"apply", "-f",
-		                       "https://raw.githubusercontent.com/slateci/federation-controller/main/resources/installation/federation-role.yaml"});
+							   "https://raw.githubusercontent.com/slateci/federation-controller/main/resources/installation/federation-role.yaml"});
```

Comment:
(Alignment should use spaces, even when leading indentation is tabs.)

Reply:
Should be fixed